### PR TITLE
Stupid oversight that Mathieu spotted

### PR DESCRIPTION
**Pull request by rtilder**
s/signature.filename/self.filename/

### CODE REVIEW

File: signing_clients/apps.py [hunk 1]
```diff
@@ -319,7 +319,7 @@ def make_signed(self, signature, outpath=None, sigpath=None):
         if os.path.exists(outpath):
             raise IOError("File already exists: %s" % outpath)
 
-        sigpath = sigpath or signature.filename
+        sigpath = sigpath or self.filename
```
Comment (andymckay) [inline signing_clients/apps.py]: right I was confused about the filename in the object, does your test catch this case?

**Comment by glasserc** [comment]
Hi, any progress on this? It looks like the same issue is still present.

**Comment by EnTeQuAk** [comment]
This is now in master and released as version 1.0 on pypi, together with all the python 3 compatibility features we recently added.

**Pull request closed without merging**